subxt/src/storage/storage_type.rs

```diff
@@ -237,6 +241,43 @@ where
             })
         }
     }
+
+    /// the storage version of a pallet.
```

Comment:
nit: I prefer upper case for documentation and for consistency, but totally up to you